360/watchapp/views.py

```diff
@@ -37,8 +179,7 @@ def register_view(request):
     error = None
 
     if request.method == 'POST':
-        first_name = request.POST.get('first_name')
-        last_name = request.POST.get('last_name')
+        fullname = request.POST['name']
```

Comment:
again, keep first name and last name, delete full name

Reply:
I dont have that in my code